e2e/fixture/socket.ts

```diff
@@ -0,0 +1,55 @@
+import { type Page } from '@playwright/test';
+import { type Address } from 'viem';
+
+import { chain } from './base';
+
+function socket(page: Page) {
+  const assets = {
+    ETH: {
+      chainId: chain.id,
+      address: '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee',
+      name: 'Ethereum',
+      symbol: 'ETH',
+      decimals: 18,
+      chainAgnosticId: null,
+      icon: 'https://maticnetwork.github.io/polygon-token-assets/assets/eth.svg',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/eth.svg',
+    },
+    OP: {
+      chainId: chain.id,
+      address: '0x4200000000000000000000000000000000000042',
+      name: 'Optimism',
+      symbol: 'OP',
+      decimals: 18,
+      chainAgnosticId: null,
+      icon: 'https://optimistic.etherscan.io/token/images/optimism_32.png',
+      logoURI: 'https://optimistic.etherscan.io/token/images/optimism_32.png',
+    },
+  };
+
+  type Balances = {
```

Comment:
minor: we generally call types in singular, so `type Balance` and instance it using `Balance[]`